Articles/Blog/2025-08-BSSwFellowsOpen2025.md

```diff
@@ -0,0 +1,86 @@
+
+- [BSSw Fellowship Program](https://bssw.io/fellowship)
+- [APPLY NOW!](https://bssw.io/pages/apply-for-the-bssw-fellowship-program)
+- <mark>Application deadline: Friday, October 31, 2025</mark>; this is a firm deadline that will not be extended.
+
+### Current and past BSSw Fellows are:
+
+- [Providing information on projects and perspectives (2024 BSSw Fellows)](https://bssw.io/blog_posts/2024-bssw-fellows-projects-and-perspectives)
+- [Advancing software productivity and sustainability via ongoing projects (2025 BSSw Fellows)](https://bssw.io/blog_posts/introducing-the-2025-bssw-fellows)
+- [Providing information on projects and perspectives (2023 BSSw Fellows)](https://bssw.io/blog_posts/2023-bssw-fellows-projects-and-perspectives)
+<!-- - [Advancing software productivity and sustainability via ongoing projects (2024 BSSw Fellows)](https://bssw.io/blog_posts/introducing-the-2024-bssw-fellows) -->
+- [Providing information on projects and perspectives (2022 BSSw Fellows)](https://bssw.io/blog_posts/2022-bssw-fellows-projects-and-perspectives)
+<!-- - [Advancing software productivity and sustainability via ongoing projects (2023 BSSw Fellows)](https://bssw.io/blog_posts/introducing-the-2023-bssw-fellows) -->
+- [Providing information on projects and perspectives (2021 BSSw Fellows)](https://bssw.io/blog_posts/2021-bssw-fellows-projects-and-perspectives)
+<!-- - [Providing information on projects and perspectives (2020 BSSw Fellows)](https://bssw.io/blog_posts/2020-bssw-fellows-projects-and-perspectives) -->
+<!-- - [Guiding developers through each stage of the scientific software lifecycle (2019 BSSw Fellows)](https://bssw.io/blog_posts/2019-bssw-fellows-guide-developers-through-each-stage-of-the-scientific-software-lifecycle) -->
+<!-- - [Tackling scientific productivity challenges (2018 BSSw Fellows)](https://bssw.io/blog_posts/2018-bssw-fellows-tackle-scientific-productivity-challenges) -->
+- And more!  [Check out how each of our BSSw Fellows are improving developer productivity and software sustainability.](https://bssw.io/pages/meet-our-fellows)
+
+### Author bio
+Elsa Gonsiorowski is Coordinator of the BSSw Fellowship Program, a member of the [PESO](https://pesoproject.org) team, and HPC I/O support specialist at [Livermore Computing, LLNL](https://hpc.llnl.gov/about-us).
+
+<br>
+
+**The BSSw Fellowship is sponsored by the U.S. Department of Energy and National Science Foundation.**
+
+<div class='fellow'>
+<div class='img_div'>
+  <img src='../../images/Logo_DOE_Unofficial_Sm.png' class='logo' />
+</div>
+
+<div class='img_div'>
+  <img src='../../images/Logo_NSF_4ColorB_Sm.png' class='logo' />
+</div>
+</div>
+
+<!---
+Publish: yes
+Track: bssw fellowship
+Pinned: no
+Topics: Funding sources and programs, projects and organizations
+RSS update: 2023-08-15
```

Comment:
Remove this

Reply:
Oops, I only intended to highlight the `RSS update` line.  That should be removed.  The rest are good.